SeleniumWebDriver/SeleniumWebDriver/LogIn-LogOut_TUTBY_PageFactory/Pages/LoginPage.cs

```diff
@@ -0,0 +1,75 @@
+﻿using System;
+using OpenQA.Selenium;
+using OpenQA.Selenium.Support.UI;
+using SeleniumExtras.PageObjects;
+
+
+namespace SeleniumWebDriver.Pages
+{
+    public class LoginPage
+    {
+        private IWebDriver _driver;
+        private WebDriverWait wait;
+        private const string URL = "https://tut.by/";
+        private const string _loginPopUpButton = "enter";
+        private const string _loginName = "login";
+        private const string _passwordName = "password";
+        private const string _loginButton = ".m-green";
+
+        //LoginPage constractor with initializing WebElements using PageFactory and navigating to URL
+        public LoginPage(IWebDriver driver)
+        {
+            _driver = driver;
+            PageFactory.InitElements(_driver, this);
+            _driver.Navigate().GoToUrl(URL);
+
+            Screenshot screenshot = ((ITakesScreenshot)_driver).GetScreenshot();
+            screenshot.SaveAsFile(@"D:\Screenshots\SeleniumTestingScreenshot.png", ScreenshotImageFormat.Png);
```

Comment:
Use independent path here because expected path will not be detected in case this project will be running on another machine. Search in Google how to save files in project root

Reply:
fixed with b8f582e